package.json

```diff
@@ -12,7 +12,7 @@
     "test": "jest",
     "test:ci:unit": "jest packages --ci --coverage",
     "test:ci:e2e": "jest e2e --ci -i",
-    "lint": "eslint --ext .js,.ts . --cache --report-unused-disable-directives",
+    "lint": "eslint --fix --ext .js,.ts . --cache --report-unused-disable-directives",
```

Comment:
Should there be yet another script with the `--fix` option?

Reply:
Please revert this `--fix` flag. It can be added in local development, but on CI it doesn't make too much sense, because in case of auto-fixable issues, it's gonna report success, the code will be changed (auto-fixed), but won't be reflected in the source control.